projects/plugins/boost/app/assets/src/css/main/variables.scss

```diff
@@ -37,7 +37,8 @@ $jetpack_green_90: #003010;
 $jetpack_green_100: #001c09;
 
 $red_50: #D63638;
-
+$orange_20: #FAA754;
+$green_40: #069e08;
```

Comment:
I don't think we need to re-define `$jetpack_green_40` as `$green_40`; we can just use `$jetpack_green_40` in `score.scss`.

Reply:
@thingalon, thank you for the improvement point. I have fixed the changes. Hopefully, it will be merged soon. :)